README.md

```diff
@@ -1,6 +1,5 @@
 <p align="center">
-    <a href="https://www.askbuddie.com">
-        <img src="https://raw.githubusercontent.com/askbuddie/readme/master/ask-buddie-icon-200x150.png" align="center" alt="askbuddie-icon"/>
+    <img src="askbuddie-bot.png" width="300">
```

Comment:
Renamed and changed background color of logo

Reply:
It's missing the `alt` attribute.

Reply:
Image is not displaying when I add alt attribute, how to fix it?

Reply:
![image](https://user-images.githubusercontent.com/39802237/121778360-4a8a5c00-cbb6-11eb-9141-4237b1ff84b3.png)